packages/superset-ui-connection/README.md

```diff
@@ -23,6 +23,7 @@ a high-level it supports:
 - supports `GET` and `POST` requests (no `PUT` or `DELETE`)
 - timeouts
 - query aborts through the `AbortController` API
+- conditional `GET` requests using `If-None-Match` and `ETag` headers
```

Comment:
I'd maybe mention `Cache` here? conditional sounds like something else to me.

Reply:
"Conditional request" is the actual name of a `GET` request with a `If-None-Match` header containing the `ETag` of the resource: https://developer.mozilla.org/en-US/docs/Web/HTTP/Conditional_requests#Conditional_headers. The fact that we're using the Cache API here is an implementation detail, we could use local storage, session storage or cookies.

Reply:
To expand on my comment: we're not simply caching, we're storing the resources and reusing them if and only if the hash hasn't changed.